content/departments/engineering/dev/tools/scaletesting.md

```diff
@@ -273,9 +273,16 @@ sg client codehost add-github \
 See https://ghe.sgdev.org/testing which is a replica of https://github.com/londonappbrewery on our GitHub instance.
 They are owned by the `testing` user, who can write on those repos.
 
-#### Organization with 200k blank repos (single initial commit with `README.md`)
+#### 200k blank repos (single initial commit with `README.md`)
+
+- 198k repositories assigned to `main-org`: https://ghe-scaletesting.sgdev.org/main-org
+  - 4750 teams have permissions on 27 or 28 repositories
```

Comment:
Can you provide the link to the teams or how someone would determine what teams these are?

Reply:
Took a while but addressed your points. Thanks!